packages/documentation/copy/en/project-config/Compiler Options.md

```diff
@@ -97,7 +97,7 @@ Option                                         | Type      | Default
 `--outDir`                                     | `string`  |                                | Redirect output structure to the directory.
 `--outFile`                                    | `string`  |                                | Concatenate and emit output to single file. The order of concatenation is determined by the list of files passed to the compiler on the command line along with triple-slash references and imports. See [output file order documentation](https://github.com/Microsoft/TypeScript/wiki/FAQ#how-do-i-control-file-ordering-in-combined-output---out-) for more details.
 `paths`<sup>[2]</sup>                          | `Object`  |                                | List of path mapping entries for module names to locations relative to the `baseUrl`. See [Module Resolution documentation](/docs/handbook/module-resolution.html#path-mapping) for more details.
-`--preserveConstEnums`                         | `boolean` | `false`                        | Do not erase const enum declarations in generated code. See [const enums documentation](https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#94-constant-enum-declarations) for more details.
+`--preserveConstEnums`                         | `boolean` | `false`                        | Do not erase const enum declarations in generated code. See [const enums documentation](https://github.com/microsoft/TypeScript/blob/master/doc/spec-ARCHIVED.md#94-constant-enum-declarations) for more details.
```

Comment:
ok, yea I mentioned that in my PR description but all the `const enums` docs are different so I wasn't sure which would be preferable

Reply:
Yeah, even this has a reasonably limited shelf life because we're re-writing the handbook for v2 - but at least it's not definitely 2+ years out of date